prompt.go

```diff
@@ -236,16 +236,24 @@ func (p *Prompt) Input() string {
 
 func (p *Prompt) readBuffer(bufCh chan []byte, stopCh chan struct{}) {
 	log.Printf("[INFO] readBuffer start")
+
+	bch := make(chan []byte)
+	go func() {
+		for {
+			if b, err := p.in.Read(); err == nil && !(b[0] == 0 && len(b) == 1) {
```

Comment:
`&& !(b[0] == 0 && len(b) == 1)`
👍 

It seems there is benefit for rendering performance on Windows.

Reply:
If `p.in.Read()` returns `[]byte{}` at the first return value without error, this evaluation order raise panic. Probably there is not such a thing, but it is more safety to evaluate with following order.

`&& len(b) == 1 && !(b[0] == 0)`